tools/visinity/vis-postprocess.xml

```diff
@@ -0,0 +1,69 @@
+<tool id="visinity_postprocess" name="Visualize Visinity Archive" tool_type="interactive" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="22.01">
```

Comment:
Interactive tools should not go into the TS (yet). Please submit them to Galaxy directly.

Maybe it would be also nice to create a `visinity.zip` datatype?

Reply:
sounds good; i selected zip on the assumption that it should be a datatype already in galaxy so admins wouldn't have to add a custom datatype, but if the IT can be added to galaxy directly, so too can the datatype definition.

two concerns:
- anyone who wants to use the tool before PR acceptance would have to pull that branch or modify galaxy manually 
- would it be backported to previous versions of galaxy?

Reply:
also, does:
> Please submit them to Galaxy directly
... mean this should NOT go in this repo? or simply that it should be omitted from TS inclusion, and still go in this repo?

Reply:
Adding a new datatype is not complicated. In your case it only changing the datatypes_conf.xml file and adding one line that subclasses from zip.

> anyone who wants to use the tool before PR acceptance would have to pull that branch or modify galaxy manually

You would need to copy this file into your Galaxy, yes. Most deployments run from a fork on the main repo eitherway, I think.

> would it be backported to previous versions of galaxy?

You could try. I don't think we have a rule for this yet.
ITs should go to the TS at some point we just never got time to finish that off.

> ... mean this should NOT go in this repo? or simply that it should be omitted from TS inclusion, and still go in this repo?

You can host it here, if you like. But it should not hit the TS. I would maintain it upstream in the Galaxy main repo.